libreoffice-core/include/LibreOfficeKit/LibreOfficeKit.hxx

```diff
@@ -486,6 +486,16 @@ public:
         return mpDoc->pClass->getCommandValues(mpDoc, pCommand);
     }
 
+    /**
+     * Forces your cursor to a given outline
+     * @param idx the id of the outline you want to navigate to
+     * @return whether going to the outline was successful or not
```

Comment:
The return type is actually the rect of the outline, not a boolean, correct?

Reply:
I'll update. In `electron-libreoffice` it's the correct return type I just forgot to update here